grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java

```diff
@@ -131,9 +134,9 @@ private ServerServiceDefinition bindInterceptors(ServerServiceDefinition service
     private void startDaemonAwaitThread() {
         Thread awaitThread = new Thread(()->{
                 try {
-                    GRpcServerRunner.this.server.awaitTermination();
+                    latch.await();
```

Comment:
While I think this makes sense, this does mean that if we ever wanted to add a feature in the future where we allow the user to manually shutdown the server in their code, we would have to remember to call `countDown` on this latch. Out of curiosity, what was the reasoning for not keeping `awaitTermination` here?